tasks1-10/task4.rb

```diff
@@ -0,0 +1,24 @@
+class Four
+  def largestPalindromeOfThreeDigits()
+    boundary = 10**3 - 1
```

Comment:
why we need this calculation? why not to use 999

Reply:
to change 3 for another num if it's necessary
 ok, I'll write code only to the specific task